### PR TITLE
[#267] Update issue templates

### LINKED ISSUE #267: Update description of issue templates
## Why

The templates in question are:
- `rfc_template.md`:
   - We are slowly moving away from RFC issues to RFC discussions, so let's update the description [(read more here)](https://github.com/nimblehq/compass/pull/1022)
- `story_template.md`:
    - We are not using this template at all, so let's remove it

## Who Benefits?

I think this is essential, as it will bring a good experience for the developers themselves

### PR DESCRIPTION
**Pull request by Tuubz**
Closes #267 

## What happened 👀

RFCs must be created through discussions instead of issues: https://github.com/nimblehq/compass/issues/963

## Insight 📝

I followed the same approach as used in Compass: https://github.com/nimblehq/compass/pull/1062

## Proof Of Work 📹

When creating issues:

|Before|After|
|------|------|
|New_Issue_·_nimblehq_compass|New_Issue_·_hoangmirs_articly|

### CODE REVIEW

**Review by luongvo — APPROVED** [review]
Awesome 🎉 I'll clone this to flutter-template 💇

**Comment by Tuubz** [comment]
@luongvo ICYDK, I created https://nimble.link/android-templates-rfc with help of the Bitly Chrome extension 😁

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+blank_issues_enabled: false
+
+contact_links:
+  - name: Request For Comments (RFC)
+    about: When having an idea on how to improve our processes, propose the idea so that the team can provide feedback.
+    url: https://nimble.link/android-templates-rfc
+
+  - name: Ask a question
+    about: When having a question about a pull request, issue, or problem, create a Q&A discussion to get help from the team.
+    url: https://github.com/nimblehq/android-templates/discussions/new?category=q-a
```
Comment (luongvo) [inline .github/ISSUE_TEMPLATE/config.yml:10]: @Tuubz Just wondering, why do you create shorten link for RFC only?

Comment (manh-t) [inline .github/ISSUE_TEMPLATE/config.yml:10]: @Tuubz why don't you create a shorten link for this URL too? 😄

**Review by manh-t — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (manh-t) [reply .github/ISSUE_TEMPLATE/config.yml:10]: It's nice to have a shortened link for QA 🙆‍♂️ In the future we don't need to update the shortened link if our discussion link update, I guess 💭

Comment (Tuubz) [reply .github/ISSUE_TEMPLATE/config.yml:10]: So we have two choices here:

A. Shorten links for both
B. Don't shorten links for both

My vote goes for `B` to keep it simple 🙏

Comment (manh-t) [reply .github/ISSUE_TEMPLATE/config.yml:10]: @Tuubz let's go with it then 🙏

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+blank_issues_enabled: false
+
+contact_links:
+  - name: Request For Comments (RFC)
+    about: When having an idea on how to improve our processes, propose the idea so that the team can provide feedback.
+    url: https://nimble.link/android-templates-rfc
```
Comment (luongvo) [inline .github/ISSUE_TEMPLATE/config.yml:6]: FYI: meanwhile in flutter_templates https://github.com/nimblehq/flutter_templates/pull/105/files

**Review by luongvo — APPROVED** [review]

**Comment by luongvo** [comment]
@Tuubz please manage to merge this PR soon, it's so complex & really needed all teammate reviews 🙏

**Comment by Tuubz** [comment]
@nimblehq/android-chapter One more approval is needed, please 🙏

**Review by sleepylee — APPROVED** [review]

**Pull request merged**